app/contributing/markdown-rules.md

```diff
@@ -134,6 +134,16 @@ the code, you can use editable placeholders.
 Editable placeholders are only supported in HTML, and are not supported with
 fenced codeblocks.
```

Comment:
While I'm at it ... (realize this isn't part of changed content, but ... I tripped over this wording. Several times, in fact ...)

```suggestion
Editable placeholders require HTML markup. They are not supported in pure markdown (fenced) codeblocks.
```